app.js

```diff
@@ -1,11 +1,11 @@
 const express = require("express");
 const app = express();
 app.use(express.json());
-const { getTopics, getReview } = require("./controllers/topicsController");
+const { getTopics, getArticleId } = require("./controllers/topicsController");

```

Comment:
would getArticleId not have it's own articleController? We would expect to separate out different data sets